cycamore_input/predicting_the_past.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" standalone="no" ?>
+
+<simulation xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
Please note that this is still a work in progress. I realized that I will be needing a script to add the reactors to the Cyclus input file other than those produced previously in order to make it work with DeployInst.

I am working on that at this moment.